packages/wallet/backend/src/app.ts

```diff
@@ -201,6 +201,18 @@ export class App {
     router.post('/quote', rafikiController.createQuote)
     router.post('/webhooks', rafikiController.onWebHook)
 
+    //payment-pointer
+    router.post(
+      '/payment-pointer/register-key',
```

Comment:
I think it make sense to change it to `/accounts/:accountId/payment-pointers/:id/register-key` so it aligns with other payment pointer endpoints